src/renderer/services/deposit/DepositTransactionService.ts

```diff
@@ -0,0 +1,92 @@
+import {Keypair as KeyPair} from "@chainsafe/bls/lib/keypair";
+import {BLSPubkey as BLSPubKey, bytes, DepositData} from "@chainsafe/eth2.0-types";
+import {createHash} from "crypto";
+import {signingRoot} from "@chainsafe/ssz";
+import abi from "ethereumjs-abi";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/mainnet";
+import {ITx} from "./types";
+import Wallet from "ethereumjs-wallet";
+import {Transaction} from "ethereumjs-tx";
+import {functionSignatureFromABI} from "./utils";
+import {EthConverter, toHexString} from "../utils/crypto-utils";
+import options from "../../../../src/renderer/services/deposit/options";
+import {depositAmountInEth, depositBLSDomain, gasLimitDepositTransaction} from "./constants";
+import {G2point} from "@chainsafe/bls/lib/helpers/g2point";
+
+/**
+ * Generate deposit params.
+ *
+ * @param signingKey
+ * @param withdrawalPubKey
+ *
+ * @return instance of ${DepositData}
+ */
+export function generateDeposit(signingKey: KeyPair, withdrawalPubKey: BLSPubKey): DepositData {
```

Comment:
could you move this to separate file. lets say utils.ts in this directory. And create index.ts that will export everything in this directory